packaging/pip_requirements.txt

```diff
@@ -1,17 +1,21 @@
 # minimal
-Pillow==8.2.0; python_version == '3.6'
-Pillow==9.0.0; python_version >= '3.7'
+# Pillow==8.1.2; python_version >= '3.7'
+Pillow==8.1.2; python_version >= '3.7' and python_version < '3.12'
+Pillow==10.3.0; python_version >= '3.12'
```

Comment:
Great! I wonder about the security implications from https://github.com/intra2net/guibot/pull/40, I guess it is not possible to update to 9.3.0 for Python 3.7?

Reply:
I have also upgrade the minimal version of Pillow for Python 3.7 from 8.1.2 to 9.3.0 as suggested by the pull you provided

Reply:
Ok, so the CI passes this way? This would be good news

Reply:
Yes! the packages doesn't run into any conflicts and they pass the CI too

Reply:
Awesome! Also resolving then